submissions/askuznetsov/html-css-popup/index.html

```diff
@@ -0,0 +1,256 @@
+                      src="./popup-icons/contacts.png"
+                      alt=""
+                      class="menu__item__link__image"
+                    />
+                    <p class="menu__item__text">Contacts</p>
+                  </a>
+                </li>
+                <li class="menu__item">
+                  <a href="#" class="menu__item__link">
+                    <img
+                      src="./popup-icons/tanslation.png"
+                      alt=""
+                      class="menu__item__link__image"
+                    />
+                    <p class="menu__item__text">Translate</p>
+                  </a>
+                </li>
+                <li class="menu__item">
+                  <a href="#" class="menu__item__link">
+                    <img
+                      src="./popup-icons/videos.png"
+                      alt=""
+                      class="menu__item__link__image"
+                    />
+                    <p class="menu__item__text">Videos</p>
+                  </a>
+                </li>
+                <li class="menu__item">
+                  <a href="#" class="menu__item__link">
+                    <img
+                      src="./popup-icons/search.png"
+                      alt=""
+                      class="menu__item__link__image"
+                    />
+                    <p class="menu__item__text">Search</p>
+                  </a>
+                </li>
+                <li class="menu__item">
+                  <a href="#" class="menu__item__link">
+                    <img
+                      src="./popup-icons/photos.png"
+                      alt=""
+                      class="menu__item__link__image"
+                    />
+                    <p class="menu__item__text">Photo</p>
+                  </a>
+                </li>
+                <li class="menu__item">
+                  <a href="#" class="menu__item__link">
+                    <img
+                      src="./popup-icons/apps.png"
+                      alt=""
+                      class="menu__item__link__image"
+                    />
+                    <p class="menu__item__text">Apps</p>
+                  </a>
+                </li>
+                <li class="menu__item">
+                  <a href="#" class="menu__item__link">
+                    <img
+                      src="./popup-icons/documents.png"
+                      alt=""
+                      class="menu__item__link__image"
+                    />
+                    <p class="menu__item__text">Document</p>
+                  </a>
+                </li>
+              </ul>
+
+              <input
+                type="checkbox"
+                name="more__checkbox"
+                id="more__checkbox"
+                class="more__checkbox"
+              />
+              <label for="more__checkbox" class="more__checkbox__label"
+                >More...</label
+              >
+
```

Comment:
You can use Prettier for formatting your files. It will make your code cleaner

Reply:
That's what Prettier recommends. I am using default configs. I didn't find any config that could resolve it.